### PR TITLE
 Issue #78, Part 2 of commit 

### PR DESCRIPTION
**Pull request by SRS-WRKS**
Some small changes to see if tests pass

### CODE REVIEW

File: database/things/abc.pldb [hunk 1]
```diff
@@ -1,7 +1,12 @@
 title ABC
 appeared 1980
 type pl
-originCommunity	Centrum Wiskunde & Informatica 
+<<<<<<< HEAD
+originCommunity Centrum Wiskunde & Informatica 
+=======
```
Comment (breck7) [inline database/things/abc.pldb:6]: what's this?

Comment (breck7) [reply database/things/abc.pldb:6]: https://www.youtube.com/watch?v=dygYh2qsx64

Comment (SRS-WRKS) [reply database/things/abc.pldb:6]: I am really sorry, i was concerned about test failure but that is never an excuse.

Comment (breck7) [reply database/things/abc.pldb:6]: Don't think. Just do.

**Pull request closed without merging**